### ISSUE TITLE
Some new-style classes are reported as old-style

### ISSUE THREAD

**Issue by pylint-bot** [pylint-dev/astroid#24]
Originally reported by: **Choongmin Lee (BitBucket: [clee704](http://bitbucket.org/clee704), GitHub: @clee704?)**

---

```
#!python

from wtforms.form import Form
from astroid import MANAGER
MANAGER.ast_from_class(Form).newstyle
```

The last line returns `False`, although `wtforms.form.Form` is a new-style class.

---
- Bitbucket: https://bitbucket.org/logilab/astroid/issue/24
